packages/jira-adapter/src/filters/account_id/account_id_filter.ts

```diff
@@ -101,6 +102,13 @@ const accountIdsScenarios = (
       callback({ value, path, fieldName })
     }
   })
+  // main scenario, sub branch of multiple account ids
+  if (Object.prototype.hasOwnProperty.call(value, ACCOUNT_IDS)
+    && isArray(value[ACCOUNT_IDS])) {
+    _.range(value[ACCOUNT_IDS].length).forEach(index => {
+      callback({ value: value[ACCOUNT_IDS], path: path.createNestedID(ACCOUNT_IDS), fieldName: index.toString() })
+    })
+  }
```

Comment:
```suggestion
  makeArray(value[ACCOUNT_IDS])
    .forEach((accountId, index) => {
      callback({ value: accountId, path: path.createNestedID(ACCOUNT_IDS), fieldName: index.toString() })
    })
```
- `makeArray` will handle what you need. `makeArray(undefined)` will simply return an empty array, so if `value[ACCOUNT_IDS]` is undefined, it should treat it as empty array and do nothing.
- The various methods of `Iterable` can optionally receive the index

Reply:
Interesting 😊 
The callback actually needs the entire array and not its members as value, so it will leave accountId to be unused and fail the build. I could not find a way to iterate only on indices of an array, so I cannot use in such a case the makeArray

Reply:
You could use `forEach((_value, index) => {})` in that case :)